src/main/resources/assets/skyblocker/lang/en_us.json

```diff
@@ -691,6 +693,19 @@
   "skyblocker.config.slayer.highlightBosses.GLOW": "Glow",
   "skyblocker.config.slayer.highlightBosses.HITBOX": "Hitbox",
 
+  "skyblocker.config.slayer.bossSpawnAlert": "Boss Spawn Alert",
+  "skyblocker.config.slayer.bossSpawnAlert.@Tooltip[0]": "\nDisplays a warning when a slayer boss spawns",
+
+  "skyblocker.config.slayer.miniBossSpawnAlert": "MiniBoss Spawn Alert",
+  "skyblocker.config.slayer.miniBossSpawnAlert.@Tooltip[0]": "\nDisplays a warning when a slayer miniboss spawns",
+
+  "skyblocker.config.slayer.slainTime": "Slayer kill time",
+  "skyblocker.config.slayer.slainTime.@Tooltip[0]": "Sends the amount of time spent to kill the slayer.",
+
+  "skyblocker.config.slayer.enableHud": "Slayer HUD",
+  "skyblocker.config.slayer.enableHud.@Tooltip[0]": "You may have to do 1 boss before all HUD elements show",
+  "skyblocker.config.slayer.slayerHud": "Slayer HUD Config...",
+
```

Comment:
Translation keys are generally sorted lexicographically, so if you can move them to their appropriate positions that would be nice.